packages/web/src/components/track/PremiumTrackSection.tsx

```diff
@@ -54,15 +53,15 @@ const messages = {
   unlocked: 'UNLOCKED',
   collectibleGated: 'COLLECTIBLE GATED',
   specialAccess: 'SPECIAL ACCESS',
-  goToCollection: 'Go To Collection',
+  goToCollection: 'Open Collection',
   sendTip: 'Send Tip',
   followArtist: 'Follow Artist',
   period: '.',
   exclamationMark: '!',
   ownCollectibleGatedPrefix:
-    'Users can unlock access by linking a wallet containing a collectible from ',
+    'Users can unlock access by linking a wallet containing a collectible from',
```

Comment:
intentional to remove space?

Reply:
Oops. In fact, no. I neglected to test the owner case for NFT collections and it definitely needs the space. Thanks for catching this.